app/views/hq/partners/index.html.erb

```diff
@@ -35,7 +35,7 @@
                 <%= partner.status.name %>
               </td>
               <td>
-                <%= partner.start_date.strftime('%B %d, %Y') %>
+                <%= partner.start_date.strftime('%d %B %Y') %>
```

Comment:
We even have a helper. It's in https://github.com/AgileVentures/osra/blob/develop/app/helpers/admin/time_helper.rb